CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ CLI:
 - `rflx setup_ide` subcommand for installing IDE integration (#795)
 - `rflx` option `--unsafe` (#987)
 
+### Removed
+
+Specification:
+
+- Remove private types (#1156)
```

Comment:
The "Remove" is redundant as it is already in the "Removed" section.

Reply:
Fair enough. Removed :wink: